migrations/2023-08-09-101305_user_instance_block/up.sql

```diff
@@ -0,0 +1,51 @@
+CREATE TABLE instance_block (
+    id serial PRIMARY KEY,
+    person_id int REFERENCES person ON UPDATE CASCADE ON DELETE CASCADE NOT NULL,
+    instance_id int REFERENCES instance ON UPDATE CASCADE ON DELETE CASCADE NOT NULL,
+    published timestamp NOT NULL DEFAULT now(),
+    UNIQUE (person_id, instance_id)
+);
+
+ALTER TABLE post_aggregates
+    ADD COLUMN instance_id integer REFERENCES instance ON UPDATE CASCADE ON DELETE CASCADE;
+
+CREATE OR REPLACE FUNCTION post_aggregates_post ()
+    RETURNS TRIGGER
+    LANGUAGE plpgsql
+    AS $$
+BEGIN
+    IF (TG_OP = 'INSERT') THEN
+        INSERT INTO post_aggregates (post_id, published, newest_comment_time, newest_comment_time_necro, community_id, creator_id, instance_id)
+        SELECT
+            NEW.id,
+            NEW.published,
+            NEW.published,
+            NEW.published,
+            NEW.community_id,
+            NEW.creator_id,
+            community.instance_id
+        FROM
+            community
+        WHERE
+            NEW.community_id = community.id;
```

Comment:
Should be okay. Still trying to dissect the reason the post_aggregates table needs an instance_id now tho.

`Community` contains the `instance_id` and that already comes back with `PostView` and `CommunityView`, so it seems like adding it again to `post_aggregates` is redundant.